x/emissions/module/rewards/scores.go

```diff
@@ -129,7 +129,7 @@ func GenerateReputerScores(
 	}
 
 	// Update topic quantile of EMA score
```

Comment:
comment is not updated

I'm also not convinced of the reasoning to use instantaneous scores over EMAs, which are more stable and less likely to be manipulated in the short term